pkgs/development/compilers/llvm/14/clang/default.nix

```diff
@@ -1,6 +1,14 @@
-{ lib, stdenv, llvm_meta
-, monorepoSrc, runCommand
-, substituteAll, cmake, libxml2, libllvm, version, python3
+{ lib
+, stdenv
+, llvm_meta
+, monorepoSrc
+, runCommand
+, substituteAll
+, cmake
+, libxml2
+, libllvm
+, version
+, python3
```

Comment:
I'm not a fan of reformatting code in commits like `llvmPackages_14: Add -lgcc to NIX_LDFLAGS for lldb`. IMO that belongs in a separate commit (especially since https://github.com/NixOS/nixpkgs/pull/201485/commits/b93b4f60855d15fc7784b5fc5c57cb0c3cd899b2 doesn't seem to make any functional changes in this file at all so maybe this change was added by accident?).

Reply:
Yes, this was added by accident. Auto-formatted file + `git commit -am ...`.

Reply:
fixed.